docs/source/CartesianConfig.rst

```diff
@@ -193,14 +196,75 @@ For example:
 
     only default
 
-Results in the following outcome:
+Results in the following:
 
 ::
 
-    name = default.three.one
-    key1 = Hello
-    key2 =
-    key3 = World
+    dict 1: default.three.one
+      key1 = Hello
+      key2 =
+      key3 = World
+
+
+2. exclude all combinations names not matching a pattern. Requires keyword 'no'
+Useful to remove particular conflicting key/value pairs from some combinations.
+
+Example::
+
+    key1 = value1
+    key2 = value2
+
+    variants:
+        - one:
+            key1 = Hello World
+            key2 = foo1
+        - two:
+            key2 = foo2
+        - three:
+
+    variants:
+        - A:
+            no one
+
+Results in the following::
+
+    Dict 1: A.two
+        key1 = value1
+        key2 = foo1
+    Dict 2: A.three
+        key1 = value1
+        key2 = value2
+
+
+3. update key/value pairs of combinations names matching a pattern
+
+Example::
+
+    variants:
+        - OpenSuse
+            initrd = b
+        - Debian
+            initrd = c
+    variants:
+        - Linux:
+            initrd = a
+
+    Linux..OpenSuse:
+    initrd = initrd
```

Comment:
If you intend to demonstrate overriding only for `Linux..OpenSuse` you need to add 4 spaces here, otherwise it overrides the `initrd` for all variants.